apps/web/src/components/icons/pdf.tsx

```
@@ -0,0 +1,20 @@
+}
+
+export function PdfIcon({ className = "", size = 20, color = "currentColor" }: PdfIconProps) {
+	return (
+		<svg
+			viewBox="0 0 384 512"
+			xmlns="http://www.w3.org/2000/svg"
+			width={size}
+			height={size}
+			className={className}
+			fill={color}
+		>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)